Firestore/Swift/Source/Codable/DocumentID.swift

```diff
@@ -130,10 +130,10 @@ public struct DocumentID<Value: DocumentIDWrappable & Codable>:
       service: "[FirebaseFirestoreSwift]",
       code: "I-FST000002",
       message: """
-      Attempting to initialize or set a @DocumentID property with a non-nil
-      value: \(value). The document ID is managed by Firestore and any
-      initialized or set value will be ignored. The ID is automatically set
-      when reading from Firestore."
+      Attempting to initialize or set a @DocumentID property with a non-nil \
+      value: \(value). The document ID is managed by Firestore and any \
+      initialized or set value will be ignored. The ID is automatically set \
```

Comment:
It's nice that this avoids the newline and indentation!

Reply:
Yes - I knew there was a way to do it, but had to search around a bit. https://www.swiftbysundell.com/tips/multiline-string-literal-tips-and-tricks/ had the info I was looking for.

Reply:
Thanks for sharing!